be/src/runtime/file_result_writer.cpp

```diff
@@ -156,9 +159,23 @@ std::string FileResultWriter::_file_format_to_name() {
 
 Status FileResultWriter::append_chunk(Chunk* chunk) {
     assert(_file_builder != nullptr);
+    int num_columns = _output_expr_ctxs.size();
+    auto result_chunk = std::make_shared<Chunk>();
+    for (int i = 0; i < num_columns; ++i) {
+        auto root = _output_expr_ctxs[i]->root();
+        if (!root->is_slotref()) {
+            return Status::InternalError("Not slot ref column");
+        }
+        ASSIGN_OR_RETURN(ColumnPtr column, _output_expr_ctxs[i]->evaluate(chunk));
+        auto column_ref = dynamic_cast<ColumnRef*>(root);
+        column = _output_expr_ctxs[i]->root()->type().type == TYPE_TIME
+                         ? ColumnHelper::convert_time_column_from_double_to_str(column)
```

Comment:
Why is the Time type treated specially here?

Reply:
> Why is the Time type treated specially here?

The TIME type is represented internally as a double and needs to be converted into a string when output externally. Here I refer to the processing logic of mysql_result_writer

Reply:
there is a converter for time type. see `csv::DatetimeConverter`